### ISSUE TITLE
item.url undefined in click-item event

### ISSUE THREAD

**Issue by jacobtender** [richardtallent/vue-simple-calendar#190]
I'd like users to be able to click calendar events and open a corresponding link. I've read the documentation, dug through the `CalendarView` code, read issues (like this [one](https://github.com/richardtallent/vue-simple-calendar/issues/60)), and I'm still coming up short. I'm able to access every item property from my `onClickItem` method, but not the url. This comes back as `undefined`. What am I missing?

```
<calendar-view
      :items="items"
      ...
      @click-item="onClickItem">
      <template #header="{ headerProps }">
          <calendar-view-header :header-props="headerProps" @input="setShowDate" />
       </template>
</calendar-view>
```
```
export default {
   name: 'app',
   data: function() {
	return { 
        ...
        items: [
          { id: 1, startDate: "2021-10-05", title: "Chroma - Cartel (2005)", url: "https://spotify.com" },
          { id: 2, startDate: "2021-10-28", endDate: "2021-10-28", title: "Cartel - Cartel (2005)", url: "https://spotify.com" },
          { id: 3, startDate: "2021-10-15", title: "Cycles - Cartel (2005)", url: "https://spotify.com" }
        ]
      }
   },
   ...
   methods: {
      onClickItem(e) {
        console.log(e.id);         // 2
        console.log(e.startDate);  // Thu Oct 28 2021 00:00:00 GMT-0400 (Eastern Daylight Time)
        console.log(e.endDate);    // Thu Oct 28 2021 00:00:00 GMT-0400 (Eastern Daylight Time)
        console.log(e.title);      // Cartel - Cartel (2005)
        console.log(e.url);        // undefined
        window.open(e.url);
      },
   }
}
```

**Comment by jacobtender** [comment]
Naturally, I find `e.originalItem.url` moments after posting. I would still expect `e.url` to work as well, however.

**Comment by richardtallent** [comment]
That's a fair expectation. I'll see if that can be added to the next point release.

**Comment by richardtallent** [comment]
Fixed in 6.0.3 (just published)!
